demo/video/src/Fabric/DemoFabric.tsx

```diff
@@ -0,0 +1,19 @@
+import { Fabric } from "@signalwire-community/react";
+import { useEffect, useState } from "react";
+
+function DemoFabric() {
+  const client = Fabric.useFabric(import.meta.env.VITE_FABRIC_TOKEN);
+  const addresses = Fabric.useAddresses(client, 10000);
+
+  return (
+    <div>
+      {import.meta.env.VITE_FABRIC_TOKEN && client && addresses !== null ? (
+        <Fabric.Video client={client} address={addresses[5]} />
```

Comment:
instead of joining `addresses[5]`, can we filter for `room` address types and join the first of them?